src/topp/IDRipper.cpp

```diff
@@ -114,9 +114,9 @@ class TOPPIDRipper :
   {
     registerInputFile_("in", "<file>", "", "Input file, in which the protein/peptide identifications must be tagged with 'file_origin'");
     setValidFormats_("in", ListUtils::create<String>("idXML"));
-    registerOutputFile_("out", "<file>", "", "The path to this file is used as the output directory.", false, false);
-    setValidFormats_("out", ListUtils::create<String>("idXML"));
-    registerStringOption_("out_path", "<file>", "", "Directory for the output files after ripping according to 'file_origin'. If 'out_path' is set, 'out' is ignored.", false, false);
+    registerOutputFile_("out", "<file>", "", "Path to the output directory to write the ripped files to.", false, false);
```

Comment:
@jpfeuffer will removing the valid format get us into trouble? @jpfeuffer

Reply:
Yes. Why was it removed? Because you can do folders? We probably need output prefix support soon.

Reply:
Cause if that is the case, I don't think it can be avoided.

Reply:
ok so do you think this can go before prefix support?

Reply:
With "output prefix" you mean all output files start with the same prefix? That would still not work with the default mode of ripper, since that extracts the original filenames from the merged idXML file.

Reply:
if we omit the file type... what will happen on KNIME side if there is no mime type associated anymore?

Reply:
It will generate an output file name without extension and pass it to the tool.
The next tool however will a) usually not accept the empty mimetype and b) not find any file, since the generated files are called different than the output name Knime generated. So you need another FileInput Node that loads the generated files based on the output prefix. Or we just correctly declare it a prefix in CTD.
A prefix could be a subfolder I think (i.e. they might not need to share anything hung in the filename).